src/Query/Filter/DueDateField.ts

```diff
@@ -11,10 +11,10 @@ export class DueDateField extends DateField {
     protected filterRegexp(): RegExp {
         return DueDateField.dueRegexp;
     }
-    protected fieldName(): string {
+    public fieldName(): string {
```

Comment:
I am conscious that this change means that the public and protected methods are a bit mixed up, when the convention of this project is:

- fields
- then public methods
- then protected methods
- then private methods

At the moment I am favouring smaller PRs where the diffs are super obvious, and re-ordering the methods would obscure the changes in this PR....